### DISCUSSION TITLE
Education pack not approved even though I have correct credentials

### DISCUSSION THREAD

**Discussion by trentmarriott808** [community/community#51622] [GitHub Education]
I was trying to get the GitHub Education pack and I got the guidance counsellor at my school to get me a digital and printed copy of a letter of enrolment that has the date, my name, and the name of the school on it. Ive tried submitting a picture of the physical copy and the digital copy both to no avail. I also tried to submit a picture of my class schedule that had dates on it and that did not work either. dose anyone have any suggestions.

**Comment by samus-aran** [comment]
Hi @trentmarriott808  👋
We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
